polyply/src/meta_molecule.py

```diff
@@ -203,6 +203,25 @@ def split_residue(self, split_strings):
         self.relabel_and_redo_res_graph(mapping)
         return mapping
 
+    def copy(self, as_view=False):
+        """
+        Overwrites the networkx copy method. This is needed to
+        correcly set all attributes.
+        """
+        if as_view is True:
+            msg = "Polyply currently does not implement copy as nx graph-views."
+            raise NotImplementedError(msg)
+        G = self.__class__(force_field=self.force_field,
+                           mol_name=self.mol_name)
+        G.molecule = self.molecule
```

Comment:
Not sure about this one. It seems more efficient to return a molecule instead of a copy of a molecule but then it really depends on wether you want to change the molecule attributes or not. Any ideas on a sensible default, perhaps have a argument like `copy_molecule=False`?

Reply:
Hmmn, I'm hesitant to add extra arguments to this method. Where is this (indirectly) called?
I would err on the side of function over speed. Wrong results are worse than slow results imo.